app/controllers/users/external_pull_requests_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Users
+  class ExternalPullRequestsController < ApplicationController
+    layout 'sidebar_metrics'
+
+    def index
+      @external_pull_requests = ExternalPullRequest
+                                .where(created_at: from.weeks.ago..Time.zone.now)
+                                .group_by { |pull_request| pull_request.owner.login }
+    end
+
+    private
+
+    def from
+      params[:from].to_i || 4
```

Comment:
i would like to move it into a service or something like that but since the query is pretty short, it goes well here in the controller, that's why i putted it there.. what do you think?

Reply:
No, I definitely prefer having it here, the closer to the origin the better.
Now that I think about it, my doubt here is about the default number chosen itself. Why 4? It sounds like a good default for the frontend, not for the backend. The backend needs a default just to avoid breaking. And a 0 would be a perfect fit for that. If we receive a request without that param, I wouldn't think of answering with a logic response.
I know I'm being picky here, so I don't mind if you don't agree 😅

Reply:
no, it has a lot of sense to me, i will change that 👍